project-canvas/src/components/StoryMapView/helpers/draggingHelpers.ts

```diff
@@ -25,8 +25,21 @@ const move = <T>(
   return { newSource: sourceClone, newDestination: destClone }
 }
 
-const isActionList = (caseId: string) => caseId.match(/^a/g)
-const isSubAction = (actionId: string) => actionId.match(/^s/g)
+const remove = <T>(list: T[], index: number) => {
+  const result = Array.from(list)
+  result.splice(index, 1)
```

Comment:
result is not corresponding to its function. we could rename it to something else

Reply:
line 29 coppies the input list and then line 30 changes the copy which is the result array. This is a generic function so better keep the names generic.